### PR TITLE
Render voting power for min dissolve delay

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

When setting the dissolve delay to the minimum on an NNS neuron (182 days and 15 hours), on the details page we say the neuron has no voting power. This is because we incorrectly used a strict inequality `>` instead of a weak inequality `>=` when comparing the dissolve delay. Coincidentally we also use language suggesting a strict inequality.

# Changes

1. Use the correct inequality in `NnsNeuronPageHeading.svelte` and `NnsNeuronVotingPowerSection.svelte`.
2. Change the language from saying "greater than" to "at least".

# Tests

1. Use strict bounds in the tests.
2. Update strings.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/lib/components/sns-neuron-detail/SnsNeuronVotingPowerSection.spec.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe("NnsStakeItemAction", () => {
     const po = renderComponent(neuronCanNotVote);
 
     expect(await po.getDescription()).toBe(
-      "The dissolve delay must be greater than 30 days, 10 hours for the neuron to have voting power. Learn more about voting power on the dashboard."
+      "The dissolve delay must be at least 30 days, 10 hours for the neuron to have voting power. Learn more about voting power on the dashboard."
```
Comment (mstrasinskis) [inline frontend/src/tests/lib/components/sns-neuron-detail/SnsNeuronVotingPowerSection.spec.ts:66]: Just to follow the pattern, maybe it makes sense to replace `-100n` with `-1n` [here (line #31)](https://github.com/dfinity/nns-dapp/pull/4963/files#diff-b43bfd66c62e1029db82fea6b5d1517cd4297ca395cd426053f6e3b739848c7cR31) as well?

**Review by mstrasinskis — APPROVED** [review]
Thank you!

File: frontend/src/tests/lib/components/sns-neuron-detail/SnsNeuronVotingPowerSection.spec.ts (hunk 1 above)
Comment (dskloetd) [reply frontend/src/tests/lib/components/sns-neuron-detail/SnsNeuronVotingPowerSection.spec.ts:68]: Yes. Thanks. Done.

Comment (mstrasinskis) [reply frontend/src/tests/lib/components/sns-neuron-detail/SnsNeuronVotingPowerSection.spec.ts:68]: Thanks!

**Pull request merged**